src/test/resources/com/puppycrawl/tools/checkstyle/checks/metrics/classfanoutcomplexity/InputClassFanOutComplexityAnnotations.java

```diff
@@ -0,0 +1,34 @@
+package com.puppycrawl.tools.checkstyle.checks.metrics.classfanoutcomplexity;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+public class InputClassFanOutComplexityAnnotations {
+
+    public void foo1(@TypeAnnotation char a) {}
+
+    public void foo2(final char @TypeAnnotation [] a){}
+
+    @MethodAnnotation
+    public void foo3(){}
```

Comment:
Please add test case for multiple annotations and annotations with parameters

Reply:
Is it fine that for such annotation:
`@TwoParametersAnnotation(value = value, tokenType = TokenTypes.ANNOTATION)`
check counts one class (and doesn't count TokenTypes)

Reply:
`TokenTypes.ANNOTATION` is just another constant reference, if such reference in method code ( for example) trigger +1 for counting it should be counted in annotation too.

Reply:
I didn't mean this.
Please create code sample like:
```
void Foo() {
int v = TokenTypes.ANNOTATION;

}
```

And look on how Check react on such code for counting

Reply:
It doesn't.

Reply:
Ok, the same to should be inside annotations

Reply:
added